update_sa_heinlein.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# Inspired by https://github.com/mailcow/mailcow-dockerized/blob/master/data/Dockerfiles/dovecot/sa-rules.sh
+
+set -e
+
+# Create temp directories
+[[ ! -d /tmp/sa-rules-heinlein ]] && mkdir -p /tmp/sa-rules-heinlein
+
+echo "Updating heinlein rules"
+# Hash current SA rules
+if [[ ! -f /etc/rspamd/custom/sa-rules ]]; then
+  HASH_SA_RULES=0
+else
+  HASH_SA_RULES=$(cat /etc/rspamd/custom/sa-rules | md5sum | cut -d' ' -f1)
+fi
+
+# Deploy
+url="http://www.spamassassin.heinlein-support.de/$(dig txt 1.4.3.spamassassin.heinlein-support.de +short | tr -d '"').tar.gz"
```

Comment:
if this is changed to:
url="http://www.spamassassin.heinlein-support.de/$(drill 1.4.3.spamassassin.heinlein-support.de txt | grep ^1.4.3.spamassassin.heinlein-support.de | cut -d\" -f2).tar.gz"

Then it works without bind tools.

Reply:
Thanks, I removed bind-tools and replaced the line with your suggestion.